components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -902,4 +908,121 @@ private static String includeData(Constants.ErrorMessage error, String data) {
         }
         return message;
     }
+
+    public JWTValidatorConfig getPrivateKeyJWTValidatorConfiguration() {
+
+        String tenantDomain = ContextLoader.getTenantDomainFromContext();
+        try {
+            if (JWTAuthenticationMgtOGSiServiceFactory.getInstance() != null) {
+
+                return new JWTValidatorConfig()
+                        .enableTokenReuse(JWTAuthenticationMgtOGSiServiceFactory.getInstance()
+                                .getPrivateKeyJWTClientAuthenticatorConfiguration(tenantDomain).isEnableTokenReuse());
+            } else {
+                throw new JWTClientAuthenticatorException(ERROR_JWT_AUTHENTICATOR_SERVICE_NOT_FOUND.message(),
+                        ERROR_JWT_AUTHENTICATOR_SERVICE_NOT_FOUND.code());
+            }
+        } catch (Exception e) {
+            if (e.getMessage().equals(ERROR_JWT_AUTHENTICATOR_SERVICE_NOT_FOUND.message())) {
```

Comment:
Swap the varaibles e.getMessage() and RROR_JWT_AUTHENTICATOR_SERVICE_NOT_FOUND.message() to avoid possible null pointers

Reply:
added with https://github.com/wso2/identity-api-server/pull/425/commits/7a512289ae94b261a68a84620d2d0be964c820ca